ui/app/AppLayouts/Chat/components/InviteFriendsPopup.qml

```diff
@@ -21,7 +21,8 @@ ModalPopup {
         anchors.left: linkText.right
         anchors.leftMargin: Style.current.smallPadding
         anchors.verticalCenter: linkText.verticalCenter
-        textToCopy: popup.getStatusText
+        textToCopy: popup.getStatusText.substr(popup.getStatusText.indexOf("https"))
+        textTooltip: "Copied \"" + popup.getStatusText.substr(popup.getStatusText.indexOf("https")) + "\" to clipboard"
```

Comment:
To enable i18n, we need to use `qsTr`. It's very simple, but I don't blame you for not knowing about it.
Here is the documentation if you want to learn about it: https://doc.qt.io/qt-5/qtquick-internationalization.html
```suggestion
        textTooltip: qsTr('Copied "%1" to clipboard').arg(popup.getStatusText.substr(popup.getStatusText.indexOf("https")))
```

Reply:
I just updated as you mentioned.